k8sbuild.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+function _k8sbuild() {
+  usage="k8build -- build, tag and push image to ecr"
+
+  REGION='eu-west-2'
```

Comment:
lower case for local/temp vars
https://stackoverflow.com/questions/673055/correct-bash-and-shell-script-variable-capitalization